core/app/models/spree/return_item.rb

```diff
@@ -222,10 +222,8 @@ def validate_no_other_completed_return_items
     def cancel_others
       Spree::ReturnItem.where(inventory_unit_id: inventory_unit_id)
                        .where.not(id: id)
-                       .where.not(reception_status: 'cancelled')
-                       .each do |return_item|
-        return_item.cancel!
-      end
+                       .not_cancelled
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 17) spaces for indenting an expression spanning multiple lines.